src/panels/config/script/ha-script-editor.ts

```diff
@@ -178,13 +181,64 @@ export class HaScriptEditor extends KeyboardShortcutMixin(LitElement) {
         .backCallback=${this._backTapped}
         .header=${!this._config?.alias ? "" : this._config.alias}
       >
+        ${this.scriptEntityId && !this.narrow
+          ? html`
+              <a
+                class="trace-link"
+                href="/config/script/trace/${this.scriptEntityId}"
+                slot="toolbar-icon"
+              >
+                <mwc-button>
+                  ${this.hass.localize(
+                    "ui.panel.config.script.editor.show_trace"
+                  )}
+                </mwc-button>
+              </a>
+            `
+          : ""}
         <ha-button-menu corner="BOTTOM_START" slot="toolbar-icon">
           <ha-icon-button
             slot="trigger"
             .label=${this.hass.localize("ui.common.menu")}
             .path=${mdiDotsVertical}
           ></ha-icon-button>
 
+          <mwc-list-item
+            graphic="icon"
+            .disabled=${!this.scriptEntityId}
+            @click=${this._showInfo}
+          >
+            ${this.hass.localize("ui.panel.config.script.editor.show_info")}
+            <ha-svg-icon
+              slot="graphic"
+              .path=${mdiInformationOutline}
+            ></ha-svg-icon>
+          </mwc-list-item>
+
+          <mwc-list-item
+            graphic="icon"
+            .disabled=${!this.scriptEntityId}
+            @click=${this._runScript}
+          >
+            ${this.hass.localize("ui.panel.config.script.picker.run_script")}
+            <ha-svg-icon slot="graphic" .path=${mdiPlay}></ha-svg-icon>
+          </mwc-list-item>
+
+          ${this.scriptEntityId && !this.narrow
+            ? html`
+                <a
+                  class="trace-link"
+                  href="/config/automation/trace/${this.scriptEntityId}"
+                  slot="toolbar-icon"
+                >
+                  <mwc-button>
+                    ${this.hass.localize(
+                      "ui.panel.config.script.editor.show_trace"
+                    )}
+                  </mwc-button>
+                </a>
+              `
+            : ""}
```

Comment:
Should be a `mwc-list-item`

Reply:
The button is included twice instead of button for desktop and list-item for mobile

Reply:
Oops